policyengine_us/parameters/gov/states/dc/tax/income/subtractions/disability_income_exclusion/cap.yaml

```diff
@@ -0,0 +1,17 @@
+description: DC caps the disability exclusion to this amount per disabled person.
+metadata:
+  unit: currency-USD
+  period: year
+  label: DC disability exclusion cap
+  reference:
+    # The legal code does not mention the specific exclusion amount
+    - title: Code of the District of Columbia § 47–1803.02. Gross income — Items included and excluded; "adjusted gross income" defined.
+      href: https://code.dccouncil.gov/us/dc/council/code/sections/47-1803.02
+    - title: 2021 DC Form D-40 Booklet
```

Comment:
I dont think I see the 15,000 or the 5,200 in the tax form either?

Reply:
I only see it in D-2440 https://otr.cfo.dc.gov/sites/default/files/dc/sites/otr/publication/attachments/2022_D-2440.pdf